src/test/java/Registration.java

```diff
@@ -0,0 +1,32 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+
+public class Registration extends BaseTest {
+    @Test
+    public static void GoingToRegistrationPage () throws InterruptedException {
+
+        WebDriver driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+
+        String url = "https://bbb.testpro.io/";
+        driver.get(url);
+
+        WebElement registrationField = driver.findElement(By.cssSelector("[id='hel']"));
+
+registrationField.click();
+
+driver.wait(5000);
```

Comment:
**Test failed!** You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/221122840-169734bb-623f-48ec-bf9e-372f349ab124.png)

**Hint**

`Thread.sleep(5000);`